app.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+'use strict';
+
+var Game = require('./lib/game.js');
```

Comment:
Any reason to use var over const?

Reply:
no, not really. I realise I should dig into the differences between `var`, `let` and `const`